### ISSUE TITLE
If a control inside a toot is focused, arrow keys should still move relative to that toot

### ISSUE THREAD

**Issue by jcsteh** [nolanlawson/pinafore#2228]
Steps to reproduce:

1. Focus the first toot in a timeline.
2. Press tab to focus a control inside it.
3. Press down arrow.
    - Expected: The second toot should be focused.
    - Actual: The first toot is focused.

This is inconvenient if:

1. A screen reader user uses their review cursor to more closely inspect the content of a toot and inadvertently focuses a control within.
2. You interact with a toot in some way that requires you to move focus inside it.

**Comment by nolanlawson** [comment]
I can repro. This looks like an oversight in how keyboard shortcuts were implemented.
